tests/service/servables/bidaf_test.py

```diff
@@ -13,27 +13,27 @@ def test_uses_named_inputs(self):
                 "passage": "One time I was writing a unit test, and it succeeded on the first attempt."
         }
 
-        bidaf_params = Params({
-                "glove_path": "tests/fixtures/glove.6B.100d.sample.txt.gz",
-                "tokenizer": {
-                        "type": "word"
-                },
-                "token_indexers": {
-                        "tokens": {
-                                "type": "single_id",
-                                "lowercase_tokens" : True
-                        },
-                        "token_characters": {
-                                "type": "characters"
+        bidaf_config = Params({
```

Comment:
Can you just load the file from `experiment_config` here, too, like you do in `servable.py`?